### PR TITLE
Upgrade paper_trail dependency to 5.x

### PR DESCRIPTION
**Pull request by travispett**
The two config lines were added to silence deprecation warnings.

https://github.com/airblade/paper_trail/blob/master/CHANGELOG.md#500-2016-05-02

### CODE REVIEW

File: lib/fast_versioning/engine.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 module FastVersioning
   class Engine < ::Rails::Engine
     isolate_namespace FastVersioning
+
+    PaperTrail.config.track_associations = false
```
Comment (iiwo) [inline lib/fast_versioning/engine.rb:5]: I don't think we should change any PaperTrail configuration in the engine, that should be done in the application - why is that needed here?

Comment (travispett) [reply lib/fast_versioning/engine.rb:5]: Thats what I thought too, but the specs were throwing warnings about this until I put this in here, even after configuring this in the setup of the dummy app for the spec.

**Comment by travispett** [comment]
Going to close these for now until i have some more time to devote to these
